readme.markdown

```diff
@@ -532,26 +573,27 @@ created instead.
 
 Other files will be copied in the same way as they are for library packages.
 
-Binary packages should not install .nim files so include
-``skipExt = @["nim"]`` in your .nimble file, unless you intend for your package to
-be a binary/library combo which is fine.
+Binary packages should not install .nim files so include ``skipExt = @["nim"]``
+in your .nimble file, unless you intend for your package to be a binary/library
+combo.
 
 Dependencies are automatically installed before building.
 It's a good idea to test that the dependencies you specified are correct by
 running by running ``nimble build`` or ``nimble install`` in the directory
 of your package.
 
-### Hybrids
```

Comment:
Why did you decide to remove this heading? I think it should stay, if only because people may have linked to it.